user-console/src/main/resources/org/pentaho/mantle/public/home/js/favorites.js

```diff
@@ -242,6 +243,23 @@ define([
       setTimeout(function () {
         that.spinner.stop();
         $("#" + that.displayContainerId).html(html);
+        $("#"+that.contentPanelId).find("a").first().attr("tabindex",0);
+        $("#"+that.contentPanelId).find("a").keydown(function (event) {
+          let nextItem;
+          let keyCode = event.which || event.keyCode;
+          if (keyCode == a11yUtil.keyCodes.arrowUp){
+            nextItem = $(this).parent().prev();
+          }else if(keyCode == a11yUtil.keyCodes.arrowDown){
+            nextItem = $(this).parent().next();
+          }else if (keyCode == a11yUtil.keyCodes.space){
+            $(this).find(".pull-right").click();
+          }
+          if ( nextItem != null && nextItem.length !== 0) {
+            $("#"+that.contentPanelId+" a[tabindex=0]").attr("tabindex",-1);
+            $(nextItem).find("a").attr("tabindex", 0);
+            nextItem.children().focus();
+          }
+        });
```

Comment:
Please, use `var` instead of `let`. Also, the formatting is off.

Reply:
Addressed.